src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -171,43 +166,42 @@ public class Elevator extends SubsystemBase implements ActiveSubsystem, Shuffleb
   public Elevator() {
     updateSensorState();
     SmartDashboard.putData("Elevator Sim", mechanism2d);
-    controller.setTolerance(GOAL_POSITION_TOLERANCE);
-  }
-
-  /** Returns elevator height. */
-  public double getHeight() {
-    return this.currentState.position;
+    controller.setTolerance(GOAL_HEIGHT_TOLERANCE);
```

Comment:
We probably don't need to set a tolerance since (I believe) this only affects `atGoal()` which we do not use.